index.html

```diff
@@ -1992,22 +1965,12 @@ <h2>Chunk naming
 <td class="Regular">Private bit: second byte</td>
 <td class="Regular">0 (uppercase) = public,<br class="xhtml" />
  1 (lowercase) = private.</td>
-<td class="Regular">A public chunk is one that is defined in this International
-Standard or is registered in the list of PNG special-purpose
-public chunk types maintained by the Registration Authority (see
-4.9 <a href="extension-and-registration"><span class=
-"xref">Extension and registration</span></a>). Applications can
-also define private (unregistered) chunk types for their own
-purposes. The names of private chunks have a lowercase second
-letter, while public chunks will always be assigned names with
-uppercase second letters. Decoders do not need to test the
-private-chunk property bit, since it has no functional
-significance; it is simply an administrative convenience to
-ensure that public and private chunk names will not conflict. See
-clause&#160;14: <a href="#editors-and-extensions"><span class="xref">Editors and
```

Comment:
Do those two linked sections no longer apply?

Reply:
They no longer apply: they discussed registration and duplicated information specified elsewhere.